src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int side1;
+    private int side2;
```

Comment:
```suggestion
    private int firstBase;
    private int secondBase;
```
avoid numbers in variable names